packages/core/content-type-builder/admin/src/utils/timeFormat.ts

```diff
@@ -0,0 +1,55 @@
+type TimeChangeHandler = (params: {
+  target: { name: string; value: string | undefined; type: string };
+}) => void;
+
+type TimeChangeParams = {
+  value: string | undefined;
```

Comment:
```suggestion
  value?: string;
```
you can do this instead just a little neater